build/utils/config.js

```diff
@@ -2,17 +2,19 @@
  * @file Provides simple user configuration options.
  */
 
-import loconfig from '../../loconfig.json';
+import loconfig from '../../loconfig.json' assert { type: 'json' };
 
 let usrconfig;
 
 try {
-    usrconfig = await import('../../loconfig.local.json');
+    usrconfig = await import('../../loconfig.local.json', {
+        assert: { type: 'json' }
+    });
     usrconfig = usrconfig.default;
 
     merge(loconfig, usrconfig);
 } catch (err) {
-    // do nothing
+    console.error(err);
```

Comment:
Does this log an error if you don't have a local config file?

Reply:
It does, good catch. I will revert the change on this line